azure/use-managed-identity.yml

```diff
@@ -0,0 +1,42 @@
+# If you don't want to use the Azure client_id/client_secret, but Azure Managed Identities instead (https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm).
+# The BOSH director VM will get a Managed Identity assigned, and with that (and the proper roles/permissions for that identity) it can do all Azure control plane actions.
+#
+- # If you use this operator, you also have to use the azure/cpi.yml operator.
+- path: /cloud_provider/properties/azure/credentials_source?
+  type: replace
+  value: managed_identity
+- path: /cloud_provider/properties/azure/default_managed_identity?/type?
+  type: replace
+  value: UserAssigned
+- path: /cloud_provider/properties/azure/default_managed_identity?/user_assigned_identity_name?
+  type: replace
+  value: ((azure-managed-identity))
+- path: /cloud_provider/properties/azure?/client_id?
+  type: replace
+  value: ""
+- path: /cloud_provider/properties/azure?/client_secret?
+  type: replace
+  value: ""
+- path: /cloud_provider/properties/azure?/tenant_id?
+  type: replace
+  value: ""
+- path: /instance_groups/name=bosh/properties/azure/credentials_source?
+  type: replace
+  value: managed_identity
+- path: /instance_groups/name=bosh/properties/azure?/client_id?
+  type: replace
+  value: ""
+- path: /instance_groups/name=bosh/properties/azure?/client_secret?
+  type: replace
+  value: ""
+- path: /instance_groups/name=bosh/properties/azure?/tenant_id?
+  type: replace
+  value: ""
+# If you also want the bosh-managed VMs to have this managed identity (can be dangerous for diego cells, since untrusted code can run there), then add these as well:
+#
+#- path: /instance_groups/name=bosh/properties/azure/default_managed_identity?/type?
+#  type: replace
+#  value: UserAssigned
+#- path: /instance_groups/name=bosh/properties/azure/default_managed_identity?/user_assigned_identity_name?
+#  type: replace
+#  value: ((azure-managed-identity))
```

Comment:
Shouldn't this be a sperate ops-file? So people can optionally include it?

Reply:
done